test/uk/gov/hmrc/helptosavefrontend/connectors/HelpToSaveConnectorSpec.scala

```diff
@@ -124,22 +124,31 @@ class HelpToSaveConnectorSpec extends TestSupport with GeneratorDrivenPropertyCh
 
           val response = EligibilityCheckResponse("HtS account already exists", 3, reasonString, 1)
 
-          mockHttpGet(eligibilityURL)(Some(HttpResponse(200, responseJson = Some(Json.toJson(response)))))
+          mockHttpGet(eligibilityURL)(Some(HttpResponse(200, responseJson = Some(Json.toJson(ECResponseHolder(Some(response)))))))
 
           val result = connector.getEligibility()
           await(result.value) shouldBe Right(AlreadyHasAccount(response))
         }
 
+      "return a None when the given nino was not found to be in receipt of tax credit" in {
```

Comment:
it's not really returning `None`, it's returning `Ineligible`

Reply:
done